index.d.ts

```diff
@@ -1034,12 +1034,12 @@ declare namespace Eris {
       reason?: string
     ): Promise<void>;
     deleteChannelPermission(channelID: string, overwriteID: string, reason?: string): Promise<void>;
-    getChannelInvites(channelID: string): Promise<ChannelInvite[]>;
+    getChannelInvites(channelID: string): Promise<GuildInviteWithMetadata[]>;
     createChannelInvite(
       channelID: string,
       options?: { maxAge?: number; maxUses?: number; temporary?: boolean; unique?: boolean },
       reason?: string
-    ): Promise<ChannelInvite>;
+    ): Promise<GuildInviteWithoutMetadata>;
```

Comment:
Discord docs don't say that invite metadata is returned on the create invite endpoint, but the previous implementation did use an interface that included it. Is the documentation accurate, or is the metadata actually returned in this case and the Discord docs are wrong? Ref https://discord.com/developers/docs/resources/channel#create-channel-invite

(this applies to all other `createInvite` methods on other structures)